litgpt/config.py

```diff
@@ -93,26 +93,19 @@ def __post_init__(self):
 
     @classmethod
     def from_name(cls, name: str, **kwargs: Any) -> Optional[Self]:
-        num_slashes = name.count("/")
-        if num_slashes == 0:
-            if name not in name_to_config:
-                # search through all `config['hf_config']['name']`
-                try:
-                    conf_dict = next(config for config in configs if name == config["hf_config"]["name"])
-                except StopIteration:
-                    return None
-            else:
-                conf_dict = name_to_config[name]
-
-        elif num_slashes == 1:
-            # get name from an org/model string like "meta-llama/Meta-Llama-3-8B"
-            org, model_name = name.split("/")
+        if name not in name_to_config:
+            # search through all `config['hf_config']['name']`
             try:
-                conf_dict = next(config for config in configs if config["hf_config"]["org"] == org and config["hf_config"]["name"] == model_name)
+                conf_dict = next(config for config in configs if name == config["hf_config"]["name"])
+
+            # try to get name from an org/model string like "meta-llama/Meta-Llama-3-8B"
             except StopIteration:
-                return None
+                try:
+                    conf_dict = next(config for config in configs if config["hf_config"]["org"] + "/" + config["hf_config"]["name"] == name)
```

Comment:
You could combine the condition with the one above, then you only need one try/except handling instead of having to nest them. 

```py
next(... if name == config["hf_config"]["name"] or name == config["hf_config"]["org"] + "/" + config["hf_config"]["name"])
```

Reply:
Great point, thanks!